src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js

```diff
@@ -85,10 +85,41 @@ class ContactForm extends Component {
             name: '',
             email: '',
             subject: '',
-            message: ''
+            message: '',
+            errors: null
         });
     }
 
+    validateForm(formValues) {
```

Comment:
Hello @YogiHa could you take a look at front-end validation? Is front-end validation identical to back-end validation?

Reply:
Actually I have left email.js dash template almost empty, so it is up to the frontend to make validations decleration,

Lovley done @aims519 :)

Reply:
Thanks, as soon as @aims519 changes var to let I will approve this PR.